packages/channel-client/tests/fakes/fake-channel-provider.ts

```diff
@@ -41,6 +42,9 @@ export class FakeChannelProvider implements ChannelProviderInterface {
       case 'PushMessage':
         return this.pushMessage(params) as Promise<MethodType[K]>;
 
+      case 'GetEthereumSelectedAddress':
+        return new Promise(() => '0xEthereumSelectedAddress') as Promise<MethodType[K]>;
```

Comment:
Couldn't this just be `return '0xEthereumSelectedAddress'`?

Reply:
Yes